internal/daysteps/daysteps.go

```diff
@@ -1,19 +1,55 @@
 package daysteps
 
 import (
-	...
+	"fmt"
+	"github.com/RavanAbbasov/go-fitness-tracker-v2/internal/personaldata"
+	"github.com/RavanAbbasov/go-fitness-tracker-v2/internal/spentenergy"
+	"strconv"
+	"strings"
+	"time"
 )
 
-const (
-	StepLength = 0.65
-)
+type DaySteps struct {
+	Steps        int
+	Duration     time.Duration
+	PersonalData personaldata.Personal
+}
+
+func (ds *DaySteps) Parse(datastring string) error {
+	parts := strings.Split(datastring, ",")
+	if len(parts) != 2 {
+		return fmt.Errorf("неверный формат данных")
+	}
+
+	steps, err := strconv.Atoi(parts[0])
+	if err != nil {
+		return fmt.Errorf("ошибка парсинга шагов: %v", err)
```

Comment:
При оборачивании нужно использовать `%w` (wrap), а просто при выводе в консоль `%v`
```suggestion
		return fmt.Errorf("ошибка парсинга шагов: %w", err)
```

Reply:
Нужно везде поправить